docs/software/prgenv/julia.md

```diff
@@ -0,0 +1,50 @@
+
+The `julia` uenv provides a complete HPC setup for running Julia efficiently at scale, using the supercomputer hardware optimally. Unlike in traditional approaches, this Julia HPC setup enables you to update Julia yourself using the included preconfigured community tool [`juliaup`](https://github.com/JuliaLang/juliaup). It also does not preinstall any packages site-wide. Instead, for HPC key packages that benefit from using locally built libraries (`MPI.jl`, `CUDA.jl`, `AMDGPU.jl`, `HDF5.jl`, `ADIOS2.jl`, etc.), this uenv provides the libraries and presets package preferences and environment variables for an automatic optimal installation and usage of these packages using these local libraries. As a result, you only need to type, e.g., `] add CUDA` in the Julia REPL, in order to install `CUDA.jl` optimally. The `julia` uenv internally relies on the community scripting project [JUHPC](https://github.com/JuliaParallel/JUHPC) to achieve this.
+
+## How to use
+
+Find and pull a Julia uenv image:
+```bash
+uenv image find julia              # list available julia images
+uenv image pull julia/VERSION:TAG  # copy version:tag from the list above
+```
+
+Start the image and activate the Julia[up] HPC setup by loading the following view(s):
+=== "`juliaup`"
+    !!! example ""
+        ```bash
+        uenv start julia/VERSION:TAG --view=juliaup
+        ```
+
+=== "`juliaup` and `modules`"
+    !!! example "This activates also modules for the available libraries like, e.g, `cuda`."
+        ```bash
+        uenv start julia/VERSION:TAG --view=juliaup,modules
+        ```
+
+There is also a view `jupyter` available, which is required for [using Julia in JupyterHub][using-julia-in-jupyterhub].
+
+!!! info "The installation of `juliaup` and the latest `julia` version happens automatically the first time when `juliaup` is called."
```

Comment:
This title is quite long and verbose.

How about "Setting up Julia the first time", with the current title "The installation of `juliaup`..." as text inside the admonition?

Reply:
done